### PR TITLE
54 hoist non react statics

### PR DESCRIPTION
**Pull request by frytyler**
This PR completes #54. 

`hoistNonReactStatics` was added below `withTracking` class. 

**Note** I branched off the prettier PR ( #53 ) so this PR will be much easier to review once that PR goes in. 

Main updates were: 

https://github.com/NYTimes/react-tracking/pull/55/files#diff-54a3b7aed7b2732b18a23f871860c7d9R197

https://github.com/NYTimes/react-tracking/pull/55/files#diff-ee27e43307e27c5f49f42c73e32110f6R116

### CODE REVIEW

**Comment by tizmagik** [comment]
I just merged the prettier PR, but there’s still a lot of diffs. Can you merge master in?

**Comment by frytyler** [comment]
Just merged master so this much easier to review now. 😄

File: src/__tests__/withTrackingComponentDecorator.test.js [hunk 1]
```diff
@@ -193,4 +193,34 @@ describe('withTrackingComponentDecorator', () => {
...
+        static someVar = 'test';
+      }
+
+      expect(StaticComponent.someMethod).toBeDefined();
+      expect(StaticComponent.someVar).toEqual('test');
+    });
+
+    it("should hoist a class's static method when wrapped via HoC", () => {
+      class StaticComponent {
+        static someMethod() {}
```
Comment (tizmagik) [inline src/__tests__/withTrackingComponentDecorator.test.js]: Can you have this return something and assert the return value?

**Comment by frytyler** [comment]
Good call that's been added.

**Pull request merged**